Grammar/python.gram

```diff
@@ -112,6 +112,7 @@ simple_stmts[asdl_stmt_seq*]:
 # will throw a SyntaxError.
 simple_stmt[stmt_ty] (memo):
     | assignment
+    | type_alias
```

Comment:
I tried this but got problems:

```
./Programs/_freeze_module importlib._bootstrap ./Lib/importlib/_bootstrap.py Python/frozen_modules/importlib._bootstrap.h
  File "<frozen importlib._bootstrap>", line 27
    return type(obj).__qualname__
           ^^^^
SyntaxError: invalid syntax
```

Reply:
Hold on, I noticed you put double quotes and I used single. Seems to work better with double quotes; I assume that indicates a soft keyword.

Reply:
Yeah, I didn't give this too much thought. There's the obvious conflict with expression where `type` by itself constitutes a valid expression. Let's leave it above `star_expressions` but include the lookahead.

Reply:
From offline discussion, this won't work because it has to be before the expression case. However, I can add a lookahead to the typealias case to improve performance. I just pushed that change.

Reply:
> I assume that indicates a soft keyword.

Oh, I've been always thinking they are the same just as in the python. Glad I've kept everything else as is in my last change.